model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/Dbsnp.java

```diff
@@ -0,0 +1,163 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.List;
+
+public class Dbsnp {
+    private String license;
+    private String alleleOrigin;
+    private List<Alleles> alleles;
+    private String alt;
+    private String chrom;
+    private String _class;
+    private Integer dbsnpBuild;
+    private List<String> flags;
+    private Gene gene;
+    private Hg19 hg19;
+    private String ref;
+    private String rsid;
+    private Boolean validated;
+    private String varSubtype;
+    private String vartype;
+
+
+    @Field(value = "license")
+    public String getLicense()
+    {
+        return license;
+    }
+
+    public void setLicense(String license) {
+        this.license = license;
+    }
+    
+    @Field(value = "allele_origin")
+    public String getAllele_origin() {
```

Comment:
@emz1003 maybe also good to remove the underscore here in the getter, better not to mix CamelCase and underscores